metric_learn/rca.py

```diff
@@ -98,26 +96,24 @@ def fit(self, X, chunks):
         When ``chunks[i] == -1``, point i doesn't belong to any chunklet.
         When ``chunks[i] == j``, point i belongs to chunklet j.
     """
+    if self.pca_comps != 'deprecated':
+      warnings.warn('"pca_comps" parameter is not used.'
+                    ' It has been deprecated in version 0.5.0 and will be'
+                    'removed in 0.6.0', DeprecationWarning)
+
     X = self._prepare_inputs(X, ensure_min_samples=2)
 
     # PCA projection to remove noise and redundant information.
-    if self.pca_comps is not None:
```

Comment:
It appears that the data preprocessing (either pca or centering), was done only at fit time, so then the fit and transform were not coherent. But now it will be. 
I'll add a `ChangedBehaviourWarning` though to ensure users know what happens